docs/sanic/config.md

```diff
@@ -53,6 +53,13 @@ import myapp.default_settings
 app = Sanic('myapp')
 app.config.from_object(myapp.default_settings)
 ```
+or also by path to config:
+
+```
```

Comment:
Would it be possible to add a sample Configuration class as well in this section so that it can be handy when one does Ctrl+C > Ctrl + V